web/web.go

```diff
@@ -526,38 +529,193 @@ func dropForeignKey(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(sessionState.conv)
 }
 
-func renameForeignKey(w http.ResponseWriter, r *http.Request) {
+func renameForeignKeys(w http.ResponseWriter, r *http.Request) {
 	table := r.FormValue("table")
-	pos := r.FormValue("pos")
-	newName := r.FormValue("name")
+	reqBody, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Body Read Error : %v", err), http.StatusInternalServerError)
+	}
+
 	if sessionState.conv == nil || sessionState.driver == "" {
 		http.Error(w, fmt.Sprintf("Schema is not converted or Driver is not configured properly. Please retry converting the database to Spanner."), http.StatusNotFound)
 		return
 	}
-	if table == "" || pos == "" || newName == "" {
-		http.Error(w, fmt.Sprintf("Table name or position or new name is empty"), http.StatusBadRequest)
+
+	changeMap := map[string]string{}
+	if err = json.Unmarshal(reqBody, &changeMap); err != nil {
+		http.Error(w, fmt.Sprintf("Request Body parse error : %v", err), http.StatusBadRequest)
+		return
+	}
+	//check new name for spanner name validity
+	newNames := getValuesAsLowerCaseSlice(changeMap)
+	if validity, invalidNewNames := checkSpannerNamesValidity(newNames); !validity {
```

Comment:
suggest:

s/validity/ok

s/invalidNewNames/invalidNames

Similar changes should also be made in `checkSpannerNamesValidity`.